tests/footer.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test } from '@playwright/test';
+import app, { pageObjects } from './helpers/pageObjects';
+
+let po: pageObjects;
+
+test.describe('Footer tests', () => {
+  test.beforeEach(async ({ page }) => {
+    po = app(page);
+    await page.goto('/');
+  });
+
+  test('Should assert visibility and url of development', async () => {
+    await po.footer.developmentUrl();
+  });
+
+  //   test('Should assert visibility and url of features', async () => {
+  //     await po.footer.featuresUrl();
+  //   });
+  // BUG DO ZGŁOSZENIA - PROWADZI DO TUTORIALS
```

Comment:
This should not be commented out, if it fails its good, after all that's why we are writing these tests for